Writing-state-migrations.md

```diff
@@ -61,14 +61,12 @@ If your PR changes the properties of an exploration or state (or other structure
 If you find new test files where changes needed to be required, try updating the list.
 
 **Links to relevant PRs:**
- - Migration related to changing state schema for all possible states: [#6249](https://github.com/oppia/oppia/pull/6249)
-
- - Migration related to changing specific interaction schema: [#6177](https://github.com/oppia/oppia/pull/6177)
- - One-off job related to migration: [#6249](https://github.com/oppia/oppia/pull/6249)
+ - Deprecate math interactions' unsupported rules and update cust arg name: [#15271](https://github.com/oppia/oppia/pull/15271)
+ - Migration related to changing specific interaction schema: [#6177](https://github.com/oppia/oppia/pull/6177). Please note that this PR does not involve changes of ```question_domain``` and ```draft_upgrade_services```.
```

Comment:
Remove this, it is quite old, maybe add some newer one (apart from the one on previous line).

Reply:
I have updated this part.